### ISSUE TITLE
About Face Frontalization

### ISSUE THREAD

**Issue by diaodeyi** [eladrich/pixel2style2pixel#66]
I have a question about "the the second change"  in the paper, how to realize “setting λ1= 0.01, λ2= 0.8 over the inner part of the face and λ1= 0.001, λ2= 0.08 elsewhere”  specifically ？i means every image is different .

**Comment by yuval-alaluf** [comment]
Here lambda1 and lambda2 correspond to the L2 loss weight and LPIPS loss weight. In frontalization, we apply a different loss weight for the inner region and outer region of the face. In the README, we provide the training arguments that you need in order to reproduce our results: 
https://github.com/eladrich/pixel2style2pixel#frontalization

**Comment by diaodeyi** [comment]
```
> loss_lpips_crop = self.lpips_loss(y_hat[:, :, 35:223, 32:220], y[:, :, 35:223, 32:220])
> loss_l2_crop = F.mse_loss(y_hat[:, :, 35:223, 32:220], y[:, :, 35:223, 32:220])
```
it seems you crop the fixed area?

**Comment by yuval-alaluf** [comment]
Yes. We crop a fixed area that approximates the central region of the image. The cropped losses are the applied to this central region.
